mu/interface/main.py

```diff
@@ -420,6 +420,12 @@ def get_save_path(self, folder):
             "Python (*.py)",
         )
         self.previous_folder = os.path.dirname(path)
+        # Ensure there's a .py extension if none is provided by the user.
+        # See issue #1571.
+        name, ext = os.path.splitext(os.path.basename(path))
```

Comment:
Just a warning that this does not work with dot files, so something like `.gitignore` will be renamed to `.gitignore.py`. We can probably consider this out of scope, but I thought it was worth pointing out.

Reply:
Good point. For safety we should probably wrap the code in a conditional like `if not name.startswith("."):`

Reply:
Yeah, sounds good 👍

Reply:
OK... fixed and tested in latest commit on this branch. Will merge once the CI turns green. :+1: Thanks for the heads-up @carlosperate